backend/objectiv_backend/schema/validate_events.py

```
@@ -38,211 +34,26 @@ def __init__(self, event_id: uuid.UUID, error_info: List[ErrorInfo]):
-
-        errors.extend(errors_context)
-    return errors
+    ignore_keys = ['_types', 'corrected_time', 'transport_time', 'collector_time']
+    event_copy = {k: v for k, v in event.items() if k not in ignore_keys}
+    config = get_collector_config()
+    headers = {'Content-type': 'Application/json'}
 
-
-def _validate_context_item(event_schema: EventSchema, context) -> List[ErrorInfo]:
-    """
-    Check that a single context has the correct attributes
-    :param context: objects
-    :return: list of found errors
-    """
-    context_type = context['_type']
-    # theoretically we could generate some json schema with if-then that we could just validate, without
-    # having to select the right sub-schema here, but that would be very complex and not very readable.
-    schema = event_schema.get_context_schema(context_type)
-    if not schema:
-        print(f'Unknown context {context_type}, ignoring')
-        return []
-    try:
-        jsonschema.validate(instance=context, schema=schema)
-    except ValidationError as exc:
-        return [ErrorInfo(context, f'context validation failed: {exc}')]
-    return []
-
-
-def _validate_event_item(event_schema: EventSchema, event) -> List[ErrorInfo]:
-    event_type = event['_type']
-    schema = event_schema.get_event_schema(event_type=event_type)
     try:
-        jsonschema.validate(instance=event, schema=schema)
-    except ValidationError as exc:
-        return [ErrorInfo(event, f'event validation failed {exc}')]
+        response = requests.post(config.schema_validation_service_url, json=event_copy, headers=headers)
 
-    return []
+        if response.status_code == 200:
+            # we have a valid response, let's check the validation result
+            result = response.json()
+            if not result['success']:
+                for e in result['error']['issues']:
+                    errors.append(ErrorInfo(data=event, info=e))
 
+    except requests.exceptions.ConnectionError as e:
```

What if it gave 500? Why not catch all exceptions? In the JS implementation you catch all.